examples/multi-agent-debate/config.py

```diff
@@ -0,0 +1,134 @@
+from rich.prompt import Prompt
+from typing import Optional
+import langroid.language_models as lm
+import langroid.utils.configuration
+from langroid.language_models import OpenAIGPTConfig
+from langroid.utils.configuration import Settings
+from generation_config_models import load_generation_config, GenerationConfig
+
+
+# Constants
+MODEL_MAP = {
+    "1": lm.OpenAIChatModel.GPT4o,
+    "2": lm.OpenAIChatModel.GPT4,
+    "3": lm.OpenAIChatModel.GPT4o_MINI,
+    "4": lm.OpenAIChatModel.GPT4_TURBO,
+    "5": lm.OpenAIChatModel.GPT4_32K,
+    "6": lm.OpenAIChatModel.GPT3_5_TURBO,
+    "7": "ollama/mistral:7b-instruct-v0.2-q8_0",
+    "8": "gemini/" + lm.GeminiModel.GEMINI_1_5_FLASH,
+    "9": "gemini/" + lm.GeminiModel.GEMINI_1_5_FLASH_8B,
+    "10": "gemini/" + lm.GeminiModel.GEMINI_1_5_PRO,
+}
+
+MISTRAL_MAX_OUTPUT_TOKENS = 16_000
+
+
+def get_global_settings(debug: bool = False, nocache: bool = True) -> Settings:
+    """
+    Retrieve global Langroid settings.
+
+    Args:
+        debug (bool): If True, enables debug mode.
+        nocache (bool): If True, disables caching.
+
+    Returns:
+        Settings: Langroid's global configuration object.
+    """
+    return langroid.utils.configuration.Settings(
+        debug=debug,
+        cache=not nocache,
+    )
+
+
+def select_model(config_agent_name: str) -> str:
+    """
+    Prompt the user to select an OpenAI or Gemini model for the specified agent.
+
+    This function prompts the user to select an option from  a list of available models.
+    The user's input corresponds to a predefined choice, which is
+    then returned as a string representing the selected option.
+
+    Args:
+        config_agent_name (str): The name of the agent being configured, used
+                                 in the prompt to personalize the message.
+
+    Returns:
+        str: The user's selected option as a string, corresponding to one of the
+             predefined model choices (e.g., "1", "2", ..., "10").
+
+    """
+    return Prompt.ask(
+        f"Select a Model for {config_agent_name}:\n"
```

Comment:
for some reason , even when I select "same LLM for all agents", it asks me twice "Select a Model for main LLM:"

Reply:
Done